snapshot.md

```diff
@@ -3,47 +3,159 @@
 layout: snapshot
 keywords:
 comments: false
-
 # Hero section
-title: Mainnet snapshots
-description: 
-
+title: Tezos snapshots for ${NETWORK}
```

Comment:
NETWORK needs to be templated. I added a file _data/tezos_metadata.json which contains just 
```json
{
  "network": "mainnet"
}
```
Zip and upload will need to write to this file. Not sure if this is the best way to get the network name in or not.